Swiift-project1/Constants.swift

```diff
@@ -17,5 +17,9 @@ struct Constants{
     static let validationRegex: String = "^[0-9]*$"
```

Comment:
global file with all constants - bad practice
its better to create separate private enum for each ViewController/Service/Models...

Reply:
isnt better for project of this size to store consts just in one file ?